lib/collector/serverless.js

```diff
@@ -243,6 +247,29 @@ class ServerlessCollector {
     }
   }
 
+  /**
+   * Tests for writing to named pipe
+   */
+  shouldUsePipe() {
+    if (this.pipeError) {
+      return false
+    }
+    return fs.existsSync(this.pipePath)
+  }
+
+  /**
+   * Writes payload to pipe
+   */
+  flushToPipe(payload) {
+    try {
+      fs.writeFileSync(this.pipePath, payload)
```

Comment:
I'm assuming we won't have any truncation concerns with this approach like we have had previously.

Reply:
I'd think we wouldn't, if the write's synchronous.